torchtune/config/__init__.py

```diff
@@ -10,8 +10,8 @@
 from ._validate import validate
 
 __all__ = [
-    "parse",
     "instantiate",
-    "validate",
+    "parse",
     "log_config",
+    "validate",
```

Comment:
I wonder if we should just put an assert at the bottom of all our `__init__.py` to ensure this remains alphabeticaly

Reply:
I'm sure this is a flake8 command somewhere - I'll investigate.